src/Controller/Admin/TargetingPageController.php

```diff
@@ -60,7 +62,7 @@ public function clearTargetingEditableDataAction(Request $request): JsonResponse
 
         $this->saveToSession($doc, $request->getSession(), true);
 
-        return $this->adminJson([
+        return $this->jsonResponse([
```

Comment:
```suggestion
        return $this->adminJson([
```
we can keep on calling adminjson here since it extends PageController. No need to use JsonHelperTrait.